### PR TITLE
edit one review of address

### PR DESCRIPTION
**Pull request by tranhuyhoangbka**

### CODE REVIEW

File: app/assets/javascripts/review.js [hunk 1]
```diff
@@ -36,4 +36,16 @@ $(document).ready(function(){
...
+  $(".icon-colspan .glyphicon-menu-up").click(function(){
+    $(this).closest(".review").find(".review-content, .list-features, .edit_review").hide()
+    $(this).hide()
+    $(this).nextAll("i").show()
+  })
+
+  $(".icon-colspan .glyphicon-menu-down").click(function(){
+    $(this).closest(".review").find(".review-content, .list-features, .edit_review").show()
+    $(this).hide()
+    $(this).prevAll("i").show()
```
Comment (samnangcattor) [inline app/assets/javascripts/review.js]: please  add ";" to  know end of the line in  javascript

File: app/views/addresses/show.js.erb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+var number_review = <%= @reviews.count %>
+$("#review-list").html("<%= j (render @reviews) %>")
+$(".next-previous-links").html("<%= j (link_to_next_page @reviews, t('Next'), class: 'btn btn-primary', remote: true) %>")
```
Comment (minhld1709) [inline app/views/addresses/show.js.erb]: chia làm 2 dòng :)

File: app/views/addresses/show.js.erb [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+$(".next-previous-links").append(" <%= j (link_to_previous_page @reviews, t('Previous'), class: 'btn btn-primary', remote: true) %>")
+if(number_review > 3){
+  $("html, body").animate({scrollTop : 900}, 1500);
```
Comment (samnangcattor) [inline app/views/addresses/show.js.erb]: remove ";" here because  it's is  file  *.js.erb

File: app/views/reviews/edit.js.erb [hunk 4]
```diff
@@ -0,0 +1,27 @@
+$("#review_<%= @review.id %>").append("<div><%= j render('form') %></div>")
```
Comment (samnangcattor) [inline app/views/reviews/edit.js.erb]: You should  change 'form' to  "form"

File: app/views/addresses/show.js.erb [hunk 5]
```diff
@@ -0,0 +1,8 @@
+var number_review = <%= @reviews.count %>
+$("#review-list").html("<%= j (render @reviews) %>")
+$(".next-previous-links").html("<%= j (link_to_next_page @reviews, t("Next"), class: "btn btn-primary", remote: true) %>")
+$(".next-previous-links").append(" <%= j (link_to_previous_page @reviews, t("Previous"), class: "btn btn-primary", remote: true) %>")
```
Comment (bapboy18) [inline app/views/addresses/show.js.erb]: so long

File: app/views/reviews/_form.html.erb [hunk 6]
```diff
@@ -0,0 +1,21 @@
+<%= form_for [@address, @review], html: {multipart: true}, remote: true do |f| %>
+  <div class="text-area-wrapper">
+    <%= f.text_area :content, class: "form-control",
+                              placeholder: t("review.placeholder") %>
+    <%= f.hidden_field :point, value: "" %>
+  </div>
+  <div class="post-action">
+    <section class="upload-img">
+      <%= image_tag Settings.review.upload_icon, title: t("review.upload_title") %>
```
Comment (daothanhcam) [inline app/views/reviews/_form.html.erb]: Break line.

File: app/views/reviews/_review.html.erb [hunk 7]
```diff
@@ -16,7 +16,7 @@
             <%= review.content %>
           </div>
           <div class="list-features">
-            <%= link_to t("edit"), "#" if can? :update, review %>
+            <%= link_to t("edit"), [:edit, @address, review], remote: true if can? :update, review %>
```
Comment (daothanhcam) [inline app/views/reviews/_review.html.erb]: Break line.

File: app/models/review.rb [hunk 8]
```diff
@@ -11,4 +11,6 @@ class Review < ActiveRecord::Base
   validates :point, presence: true
 
   mount_uploader :image, PhotoUploader
+
+  paginates_per Settings.pages.ten
```
Comment (dieunb) [inline app/models/review.rb]: Settings.pagination.per_page

**Pull request merged**